File: DecimaI Comparator/DecimaIComparator.cs
namespace CSharp_problems
{
    class MainClass
    {
        public static void Main(String[] args)
        {
            Console.WriteLine(areEqualByThreeDecimalPlaces(3.123,3.1234));
        }
        public static Boolean areEqualByThreeDecimalPlaces(double a, double b)
        {
            a = a * 1000;
            b = b * 1000;
            if ((int)(a) == (int)(b))
                return true;
            else
                return false;
        }
    }
}

File: DecimaI Comparator/DecimaIComparator.swift
import UIKit

func areEqualByThreeDecimalPlaces(a:Double, b:Double)->Bool{
    let a = a*1000;
    let b = b*1000;
    if((Int) (a) == (Int) (b)){
        return true;
    }
    else{
        return false;
    }
}
print(areEqualByThreeDecimalPlaces(a: 1.234, b: 1.23456))
